src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.ValueGenerator/Main.cs

```diff
@@ -110,7 +130,33 @@ public List<Result> Query(Query query)
             }
             catch (FormatException e)
             {
-               Log.Debug(GetTranslatedPluginTitle() + ": " + e.Message, GetType());
+                Log.Debug(GetTranslatedPluginTitle() + ": " + e.Message, GetType());
+                if (string.IsNullOrEmpty(query.ActionKeyword))
+                {
+                    return results;
+                }
+
+                return GetSuggestionFuzzyResults(query, results);
```

Comment:
I suggest put the "get suggestions" code into the if and switch to negated (not keyword empty). Then you can remove the `return results;` and we have less duplicate code.

Reply:
Got it, thank you.